runtime/src/identity.rs

```diff
@@ -793,22 +748,25 @@ decl_module! {
                                     T::AcceptTransferTarget::accept_ticker_transfer(did, auth_id),
                                 AuthorizationData::TransferTokenOwnership(_) =>
                                     T::AcceptTransferTarget::accept_token_ownership_transfer(did, auth_id),
+                                AuthorizationData::AddMultisigSigner =>
+                                    T::AddSignerMultisigTarget::accept_multi_sig_signer(Signer::from(did), auth_id),
                                 _ => Err("Unknown authorization")
                             };
                         }
                     }
                 },
-                Signer::Key(_key) => {
+                Signer::Key(key) => {
                     for auth_id in auth_ids {
                         // NB: Even if an auth is invalid (due to any reason), this batch function does NOT return an error.
                         // It will just skip that particular authorization.
                         if <Authorizations<T>>::exists((signer, auth_id)) {
-                            let _auth = Self::authorizations((signer, auth_id));
-                            // NB: Result is not handled, invalid auths are just ignored to let the batch function continue.
-                            // TODO: Add cases where signing key is authorized
-                            // let _result = match auth.authorization_data {
-                            //     _ => Err("Unknown authorization")
-                            // };
+                            let auth = Self::authorizations((signer, auth_id));
+                            //NB: Result is not handled, invalid auths are just ignored to let the batch function continue.
+                            let _result = match auth.authorization_data {
```

Comment:
Perhaps you could later iterate over the results to compute something interesting?

Reply:
I could but I'm not sure what. All of the functions being called in the match statement emit proper events on success/failure so we can track what auth which auth went through and which didn't that way.

Reply:
Would it be OK to collect results in the for-loop and then check if there exist errors and return an `Err` from `batch_accept_authorization` if they do?

Or is the intention here to mute unknown authorisations?

Reply:
When a transaction returns `Err` in our system, it signifies that ALL actions failed. I am muting unknown authorizations here because I think that's the best way to handle these batch transactions.

Reply:
See my comment below.